common/lib/xmodule/xmodule/css/problem/edit.scss

```diff
@@ -110,40 +112,33 @@
   width: 26px;
   height: 21px;
   vertical-align: middle;
-  background: url('#{$static-path}/images/problem-editor-icons.png') no-repeat;
-}
+  color: $base-font-color;
 
-.problem-editor-icon.heading1 {
-  width: 18px;
-  background-position: -265px 0;
-}
+    &.heading3 {
 
-.problem-editor-icon.multiple-choice {
-  background-position: 0 0;
-}
+    }
 
-.problem-editor-icon.checks {
-  background-position: -56px 0;
-}
+    &.multiple-choice {
 
-.problem-editor-icon.string {
-  width: 28px;
-  background-position: -111px 0;
-}
+    }
 
-.problem-editor-icon.number {
-  width: 24px;
-  background-position: -168px 0;
-}
+    &.checks {
 
-.problem-editor-icon.dropdown {
-  width: 17px;
-  background-position: -220px 0;
-}
+    }
 
-.problem-editor-icon.explanation {
-  width: 17px;
-  background-position: -307px 0;
-}
+    &.string {
```

Comment:
These are here as placeholders while UX reviews the icons and they're styling. Any unnecessary rules will be removed prior to merging.